documentation/manage/updating-your-extension.md

```diff
@@ -0,0 +1,216 @@
+
+<!-- Content with Table of Contents Module -->
+
+{% capture content_with_toc %}
+
+Firefox supports automated updates to add-ons using JSON update manifests. Add-ons hosted on [addons.mozilla.org](https://addons.mozilla.org/) (AMO) automatically receive updates to new versions posted there. Other add-ons must specify the location of their update manifests.
+
+<!-- Note -->
+
+{% capture note %}
+
+You must host your update manifest file on a secure (HTTPS) server.
+
+{% endcapture %}
+{% include modules/note.html
+	content=note
+	alert=false
+%}
+
+<!-- END: Note -->
+
+A typical update manifest looks something like:
+
+<!-- Syntax Highlighting -->
+
+{% highlight json linenos %}
+
+{
+"addons": {
+"{abcd1234-1abc-1234-12ab-abcdef123456}": {
+"updates": [
+{ "version": "0.1",
+"update_link": "https://example.com/addon-0.1.xpi" },
+{ "version": "0.2",
+"update_link": "http://example.com/addon-0.2.xpi",
+"update_hash": "sha256:fe93c2156f05f20621df1723b0f39c8ab28cdbeec342efa95535d3abff932096" },
+{ "version": "0.3",
+"update_link": "https://example.com/addon-0.3.xpi",
+"applications": {
+"gecko": { "strict_min_version": "44" } } }
+]
+}
+}
+}
+
+{% endhighlight %}
+
+<!-- END: Syntax Highlighting -->
+
+## Enabling updates to your extension
+
+If your extension is not hosted on AMO, you must specify the location of your update manifest in your extension. For extensions developed with WebExtension APIs, add the following to your manifest:
+
+<!-- Syntax Highlighting -->
+
+{% highlight json linenos %}
+
+"applications": {
+"gecko": {
+"update_url": "https://example.com/updates.json"
+}
+}
+
+{% endhighlight %}
+
+<!-- END: Syntax Highlighting -->
+
+For XUL add-ons, add the following to the `<Description about="urn:mozilla:install-manifest">` element of your `install.rdf` file:
+
+<!-- Syntax Highlighting -->
+
+{% highlight XML linenos %}
+
+<em:updateURL>https://example.com/updates.json</em:updateURL>
+
+{% endhighlight %}
+
+<!-- END: Syntax Highlighting -->
+
+## Manifest Structure
+
+The manifest is a JSON file, with a top-level object literal. This object may have the following properties:
+
+<!-- Table -->
+
+{% capture table %}
+
+| Property | Type     | Description                                                                                                                                                                                                       |
+| -------- | -------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `addons` | `object` | An object containing one entry for each add-on to be updated. For each such entry, the name of the property must be the add-on's UUID, and the value must be an [object describing the add-on]() and its updates. |
+
+{% endcapture %}
+{% include modules/table.html
+	content=table
+%}
+
+<!-- END: Table -->
+
```

Comment:
Need an h3 (###) for `add-on objects` here and the highlighted `add-ons []*` beneath it